### PR TITLE
Add script that checks concepts

### PR DESCRIPTION
**Pull request by angelikatyborska**
We need to fix two concepts:

```
pattern-matching ⨯
  Too many exercises practice this concept.
  Practiced by (11): [alphametics, bowling, meetup, perfect-numbers, poker, pov, protein-translation, queen-attack, tournament, triangle, wordy]


recursion ⨯
  Too many exercises practice this concept.
  Practiced by (11): [accumulate, binary-search, binary-search-tree, matching-brackets, pascals-triangle, pov, prime-factors, roman-numerals, sieve, spiral-matrix, strain]

```

### CODE REVIEW

**Comment by neenjaw** [comment]
Awesome! Love the Mix.install,  Could make those practice changes here to get it passing before merge.

File: bin/check_concepts.exs [hunk 1]
```diff
@@ -0,0 +1,128 @@
+Mix.install([
+  {:jason, "~> 1.2"}
+])
```
Comment (neenjaw) [inline bin/check_concepts.exs:3]: 😍

**Review by neenjaw — APPROVED** [review]

**Comment by angelikatyborska** [comment]
Ok, here are my proposed changes:
- POV doesn't practice pattern matching, only recursion
- Accumulate doesn't practice recursion, only lists and anonymous functions

I don't have a great justification for this choice... my only goal was to avoid exercises that don't practice anything (so for example I wouldn't remove recursion from Roman Numerals, because then Roman Numerals don't practice anything).

**Comment by neenjaw** [comment]
I think this is a product problem that will have to be addressed at somepoint because having more exercises in a track means these collisions will happen more

**Comment by angelikatyborska** [comment]
Oh, we need one more change after adding Satellite:
- Prime Factors no longer practice recursion because they're more specific than that, they practice tail-call recursion

**Pull request merged**